libs/product/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "@daffodil/product",
+  "version": "1.1.0-a",
+  "description": "Interfaces built for daffodil/state",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/graycoreio/daffodil"
+  },
+  "author": "Graycore LLC",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/graycoreio/daffodil/issues"
+  },
+  "scripts": {
+    "build": "ng build product",
+    "lint": "ng lint product",
+    "test": "ng test product --watch=false --browsers=ChromeHeadless"
```

Comment:
Let's leave this out temporarily until we have something to test.

Reply:
removed